src/test/regress/input/multi_behavioral_analytics_create_table_shadow.source

```diff
@@ -0,0 +1,399 @@
+--
```

Comment:
Similar to the comment for multi_test_helpers, (1) add comment why this is needed, (2) use `\include` to avoid repetitions.